data/dbus/io.snapcraft.Launcher.service.in

```diff
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=io.snapcraft.Launcher
+Exec=@bindir@/snap userd
```

Comment:
Feels like libexecdir thing to me.

Reply:
This is the regular `snap` binary that provides the userd subcommand. So we need bindir here unless I miss something.

Reply:
This is correct, though I expected this to be implemented in a libexecdir command, as users shouldn't *usually* do things here. Not a biggie, though.